Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData/Airport.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Ibistic.Public.OpenAirportData
+{
+    public sealed class Airport
```

Comment:
It may make sense to make the data types serializable, in case someone wants to transport or store them as state.